scripts/wsrep_sst_clone.sh

```diff
@@ -520,9 +520,9 @@ then
 
     wsrep_log_debug "-> WSREP_SST_OPT_LPORT: $WSREP_SST_OPT_LPORT "
     wsrep_log_debug "-> MYSQL_ACLIENT: $MYSQL_ACLIENT "
-    wsrep_log_debug "-> MYSQL_ACLIENT ADMIN_PSWD: $ADMIN_PSWD"
+    #wsrep_log_debug "-> MYSQL_ACLIENT ADMIN_PSWD: $ADMIN_PSWD"
```

Comment:
remove

Reply:
@kamil-holubicki (valid for this and all the following) I did not remove them on purpose.
One thing is to do not print them out (also if debug) another is to remove the.
In case of deep debug having there already the debug string can significantly speed-up the problem identification and resolution.
As comment in a bash there is no additional risk of exposure as if you expose the variable.

Reply:
Not a critical thing. I just try to avoid leaving 'handy things that could potentially be useful". If someone debugs and needs this info, he can add the needed debug print. 
You can leave it as it is, or remove. Up to you.